sql/catalyst/src/main/java/org/apache/spark/sql/connector/catalog/SupportsNamespaces.java

```diff
@@ -72,8 +67,7 @@
    *
    * They need specific syntax to modify
    */
-  List<String> RESERVED_PROPERTIES =
-    Arrays.asList(PROP_COMMENT, PROP_LOCATION, PROP_OWNER_NAME, PROP_OWNER_TYPE);
+  List<String> RESERVED_PROPERTIES = Arrays.asList(PROP_COMMENT, PROP_LOCATION, PROP_OWNER);
```

Comment:
not related to this PR. I think which properties are reserved is an internal thing. We should put this list in an internal class like `CatalogV2Utils`, not in a public interface.

Reply:
sgtm,